app/retail/templates/emails/grants/thank_you_for_supporting.html

```diff
@@ -114,7 +114,7 @@ <h2 style="text-transform: none;">{% trans "The world of open source is a better
 <hr>
 <div class="grantInfo">
   <a href="{% url 'grants:details' grant.id grant.slug %}">
-    <img src="{% if grant.logo and grant.logo.url %}{{ grant.logo.url }}{% else %}{% static 'v2/images/bitmap.png' %}{% endif %}" width="50" height="50" />
+    <img src="{% if grant.logo and grant.logo.url %}{{ grant.logo.url }}{% else %}{% with grant_logo='v2/images/grants/logos/' id=grant.id|modulo:3|add:1 %} {% static grant_logo|addstr:id|add:'.png' %} {% endwith %} {% endif %}" width="50" height="50" />
```

Comment:
One downside to this approach over using `static()` in the template tag is that we no longer serve the hashed asset.  I'm not too concerned with this, since it's doubtful these will change, but just something to consider!